infrastructure/resource_group_level/modules/vnet.bicep

```diff
@@ -29,7 +29,7 @@ var appServicePlanEgressSubnetName = '${appServicePlanName}-app-service-plan-egr
 var gatewaySubnetName = '${environment}-gateway'
 var privateEndpointsSubnetName = '${environment}-private-endpoints'
 
-var aciSubnetName = '${environment}-aci'
+var acaSubnetName = '${environment}-aca'
```

Comment:
I'm not certain of the best naming here - "aca" for azure container app environment seems ok. We could call it clamav, but I suppose the vnet doesn't care what's inside, just that it is an ACA environment.